jsconfig.json

```diff
@@ -1,12 +1,14 @@
 {
 	"compilerOptions": {
+		"checkJs": true,
 		"allowSyntheticDefaultImports": true,
 		"module": "commonjs",
 		"target": "ES6",
 		"baseUrl": ".",
 		"paths": {
 			"*": [ "test/*", "server/*", "client/*" ]
-		}
+		},
+		"jsx": "react"
```

Comment:
i thought this just affected how transpilation worked? does it have other effects?